lib/config/v2/gl_eink.ex

```diff
@@ -12,7 +12,8 @@ defmodule ScreensConfig.V2.GlEink do
           alerts: Alerts.t(),
           line_map: LineMap.t(),
           evergreen_content: list(EvergreenContentItem.t()),
-          audio: Audio.t()
+          audio: Audio.t(),
+          platform_location: :front | :back
```

Comment:
Might want to add `| nil` if this can be nil—looks like that's the default value if not specified in the JSON.